src/TestInfrastructure/TestInfrastructure.ruleset

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="SonarQube" ToolsVersion="14.0">
```

Comment:
Have quite a few of COM interfaces implemented and this issue is being raised due to the high number of arguments.